teuthology/suite/placeholder.py

```diff
@@ -57,6 +57,13 @@ def _substitute(input_dict, values_dict):
         'admin_socket': {
             'branch': Placeholder('ceph_branch'),
         },
+        'ceph_ansible': {
```

Comment:
I can move them to init if it makes more sense there,  we have been using placeholder for defaults so added them there.